sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/blob_options.hpp

```diff
@@ -153,7 +153,7 @@ namespace Azure { namespace Storage { namespace Blobs {
   };
 
   /**
-   * @brief Optional parameters for BlobServiceClient::ListBlobContainers.
+   * @brief Optional parameters for #Azure::Storage::Blobs::BlobServiceClient::ListBlobContainers.
```

Comment:
If the type is inside the same namespace we don't need to add the full namespace path, doxygen will resolve it.
However, that's a nit

Reply:
I tried, it didn't work